recommendations/index.md

```diff
@@ -135,6 +135,50 @@ Furthermore, multiple filters can be applied to a single request:
 GET /comments?filter[post]=1,2&filter[author]=12 HTTP/1.1
 ```
 
+## <a href="#including-links" id="including-links" class="headerlink"></a> Including Top-level, Resource-level and Relationship Links
+
+The base specification is agnostic about including links with a resource response. However, it is recommended that the following links be included within response documents:
+
+- **Top-level links** like a self-link (for the whole response) as well as relative pagination links (if appropriate).
+- **Resource-level links** like a self-link for each resource (which differs from the top-level, if the resource is part of a collection).
+- **Relationship links** for all available relationships of a resource.
+
+For example, the following request response for a comment returns all recommended links:
```

Comment:
I think "request response" sounds awkward here. How about "For example, a request for a collection of comments could prompt the following response:"